AsyncDisplayKit/Layout/ASLayout.mm

```diff
@@ -138,6 +138,12 @@ + (instancetype)layoutWithLayout:(ASLayout *)layout position:(CGPoint)position
                                sublayouts:layout.sublayouts];
 }
 
+- (NSString *)description
+{
+    return [NSString stringWithFormat:@"<<ASLayout: %p>, position = %@; size = %@; constrainedSize = %@",
+            self, NSStringFromCGPoint(self.position), NSStringFromCGSize(self.size), NSStringFromCGSize(self.constrainedSize)];
```

Comment:
@Adlai-Holler I was trying to make it look like a UIView description. Do you have a strong preference here?